spec/features/applications/new_spec.rb

```diff
@@ -22,6 +22,121 @@
     @bloom = @fancy.pets.create(name: 'Irishman', breed: 'irish wolfhound', age: 2, adoptable: true)
   end
   
+  describe '2. Starting an Application' do
+    describe "the pet index page has link to 'Start an Application'" do
+      it 'takes the user to the new application page' do
+        visit pets_path
+
+        click_link ('Start an Application')
+        
+        expect(current_path).to_not eq(pets_path)
+        expect(current_path).to eq(applications_new_path)
+      end
+    end
+    
+    describe 'the new application page has a form, a submit link, and a cancel link' do
+      before(:each)do
+        visit applications_new_path
+      end
+
+      it "has 'Cancel' link that returns user to the pets index" do      
+        click_link ('Cancel')
+
+        expect(current_path).to_not eq(applications_new_path)
+        expect(current_path).to eq(pets_path)
+      end
+
+      it 'has a field for all the attributes on the form' do
+  
+        expect(page).to_not have_field(:status)
+
```

Comment:
The status will need to be updated for this entry in the table, consider a hidden field for status and a default value.